dashboard/test/controllers/teacher_feedbacks_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class TeacherFeedbacksControllerTest < ActionController::TestCase
```

Comment:
Since this is a new test, it should follow the [new guidelines](http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers) and derive from `ActionDispatch::IntegrationTest` rather than `ActionController::TestCase`.